cg/meta/workflow/tower_common.py

```diff
@@ -0,0 +1,27 @@
+"""Module for Tower Analysis API."""
```

Comment:
Regarding the file names (`tower_common.py` and `nextflow_common.py`): I think it implies that they are the parent API so i would leave them just as tower.py or nextflow.py. And then we can have specific ones tower_balsamic.py for example

Reply:
I don't think there will be specific files for different workflows.  Common things will go to tower_common or nextflow_common and the rest within the workflow (balsamic, rnafusion, etc). In that case, do you still think it should be named `tower.py` and `nextflow.py`?

Reply:
Then I don't think they belong in `workflows`, and the class names are not consistent with what we have in place: `NextflowAnalysisAPI` isn't actually an `AnalysisAPI`, is more of a handler or of an app, don't you think? I would create a new folder for them, and then you no longer need to have common in the file names

Reply:
As we discussed this will go in a different PR, for which we might need a refinement session

Reply:
👍